lesson07/assignment/spaceland/src/components/AttractionsPage.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+<div class="attractions">
+  <h2 class="text-danger text-center m-4 p-5">{{ msg }}</h2>
+
+  <body>
+    <main class="main">
+      <div class="d-flex flex-wrap justify-content-center">
+        <img class="img-fluid img-thumbnail m-2 w-25" v-for="image in images" v-bind:key="image" :src="image.url" :alt="image.alt" />
```

Comment:
This is creating errors in the console. You are assigning the key of the image to be an image object. You'll want to make sure your keys are only being set to integers or strings. An easy fix would be adding an "id" property to your image data. Then you'd do something like: `v-bind:key="image.id"`

Reply:
Thanks! I fixed it.